mobile/lib/models/air_quality_reading.dart

```diff
@@ -43,11 +42,14 @@ class AirQualityReading extends HiveObject with EquatableMixin {
       throw Exception("pm2.5 is null for site ${site.getName()}");
     }
     List<HealthTip> healthTips = [];
-
-    for (final healthTip in json['health_tips']) {
-      try {
-        healthTips.add(HealthTip.fromJson(healthTip));
-      } catch (_, __) {}
+    dynamic jsonHealthTips = json['health_tips'];
```

Comment:
Thanks @NoahNsimbe , I hope that we are now using the dynamic health tips (from Events endpoint) which @BenjaminSsempala  created sometime back. Thanks!

Reply:
Here are the tests for health tips.
https://github.com/airqo-platform/AirQo-frontend/pull/1151#discussion_r1179957229